test_util/summary.cpp

```diff
@@ -0,0 +1,123 @@
+//#include <chrono>
+#include <iomanip>
+#include <iostream>
+#include <tuple>
+#include <getopt.h>
+#include <sstream>
+  
+#include <opm/io/eclipse/ESmry.hpp>
+
+
+static void printHelp() {
+
+    std::cout << "\ngetSmry needs minimum two argument. First is smspec filename and then list of vectors  \n"
```

Comment:
getSmry is in fact a better, less generic name for the binary rather than 'summary'

Reply:
on alternaive is maybe summaryEcl to be in line with the others ?

Reply:
summary is available, why not take it? Compare was taken by imagemagick, that's why the awkward ECL is there.

Reply:
did not know it was a contest ;) i thought it was about making it clear to the user what a binary does.

Reply:
Users think of the files as summary files. What is clear from a programmer is often not clear for the user.  For instance, the application for commandline plotting is summaryplot, not summaryEclPlot. What is a good variable naming convention for a programmer is often not a good naming convention for user applications.

Reply:
but but but then users are wrong! ;p i really don't care, keep it as summary then.

Reply:
Cool, thanks! This one is not a biggie for me either, but I do think it was super good that we chose the generic name Flow a few years back.